ykcompile/src/lib.rs

```diff
@@ -160,6 +180,49 @@ impl TraceCompiler {
         Ok(())
     }
 
+    fn c_call(
+        &mut self,
+        op: &CallOperand,
+        args: &Vec<Operand>,
+        dest: &Option<Place>,
+    ) -> Result<(), CompileError> {
+        // FIXME find a better way to get rid of the unneeded call to stop_tracing
+        match op {
+            ykpack::CallOperand::Fn(s) => if s.contains("stop_tracing") { return Ok(()) },
+            ykpack::CallOperand::Unknown => {}
+        };
+        // Save all registers to the stack
+        self.store_registers();
+        // Move call arguments into registers.
+        for (i, op) in args.iter().enumerate() {
+            let reg = Local((i + 1) as u32);
```

Comment:
It seems like we are bypassing the register allocator here. Shouldn't we use `local_to_reg()`?

Reply:
We are not, since `mov_local_local`, etc., use the register allocator, but I agree that the variable `reg` is wrongly named here, and should instead be called `local`.

Reply:
ah! I see.

Yes, please rename.